umccrise/__init__.py

```diff
@@ -339,7 +339,7 @@ def prep_stages(run, include_stages=None, exclude_stages=None):
     default_enabled = {
         'conpair',
         'structural',
-        'somatic', 'germline', 'maf',
+        'somatic', 'germline', 'maf', 'pierian',
```

Comment:
Don't need to have `pierian` here - `-S pierian`/`-E pierian` options won't work anyway, as you have it unconditionally under "somatic" rule, and that's fine!